<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Properties</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon/resources.qrc">
    <normaloff>:/images/ico.png</normaloff>:/images/ico.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Syntax Highlighting</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="syntax_radio1">
            <property name="text">
             <string>Use Built-in</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="syntax_radio2">
            <property name="text">
             <string>Use Pygments</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Pygments Style</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="pygments_style">
            <item>
             <property name="text">
              <string>default</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>emacs</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>friendly</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>friendly_grayscale</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>colorful</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>autumn</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>murphy</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>manni</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>material</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>monokai</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>perldoc</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>pastie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>borland</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>trac</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>native</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>fruity</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>bw</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>vim</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>vs</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>tango</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>rrt</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>xcode</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>igor</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>paraiso-light</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>paraiso-dark</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>lovelace</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>algol</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>algol_nu</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>arduino</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>rainbow_dash</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>abap</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>solarized-dark</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>solarized-light</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sas</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>stata</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>stata-light</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>stata-dark</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>inkpot</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>zenburn</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gruvbox-dark</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gruvbox-light</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>dracula</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>one-dark</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>lilypond</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Aspect Ratio Indicator</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="aspect_ratio_show">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Show</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="pdflatex_edit">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Target Screen Ratio</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="aspect_ratio_over">
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>11</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>13</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>17</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>18</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="aspect_ratio_under">
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>11</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>13</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>17</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>18</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Pdflatex</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pdflatex"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>PDF to PNG Converter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pdf2png"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon/resources.qrc"/>
 </resources>
 <connections/>
</ui>
